### PR TITLE
Fix NVIDIA GPUs detection

### PR DESCRIPTION
**Pull request by ventureoo**

### CODE REVIEW

File: scripts/nvidia-pci-ids-dumper/nvidia-pci-ids-dumper [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+            end
+        end
+    end
+
+    return drivers
+end
+
+local function main()
+    if #arg < 1 then
+        err("Specify path to the supported-gpu.json file")
```
[the remark this answers is not in the document]
Comment (ventureoo) [reply scripts/nvidia-pci-ids-dumper/nvidia-pci-ids-dumper:80]: It comes bundled with the NVIDIA driver. The Ids in this PR are taken from NVIDIA-Linux-x86_64-560.31.02/supported-gpus/supported-gpus.json.

[the remark this answers is not in the document]
Comment (vnepogodin) [reply scripts/nvidia-pci-ids-dumper/nvidia-pci-ids-dumper:80]: Oh alright

**Review by vnepogodin — APPROVED** [review]
LGTM

**Comment by vnepogodin** [comment]
we can land ids for `nvidia-dkms` after that PR actually

**Comment by ventureoo** [comment]
> we can land ids for `nvidia-dkms` after that PR actually

Done

**Review by vnepogodin — APPROVED** [review]
good-ich

**Pull request merged**